### DISCUSSION TITLE
Small webpage search bar can invisibly activated with /

### DISCUSSION THREAD

**Discussion by PowerUser64** [pradyunsg/furo#512] [Potential Issue] [answered]
If you go to a website using this theme (ex: https://manual.zrythm.org/) and reduce the horizontal size of your window, the sidebar on the left that contains the search bar will collapse. This is fine and expected. From here, the search bar can still be focused by pressing the <kbd>/</kbd> key. This is also fine, but the real problem is that the search bar is remains hidden in the side menu. To solve this, I would suggest expanding the collapsed sidebar with the search bar or adding another search bar that appears at the top of the page when the <kbd>/</kbd> key is pressed.

**Comment by pradyunsg** [comment, marked as answer]
Thanks, that's actually a great suggestion.

Would you be willing to file a [trackable change issue](https://github.com/pradyunsg/furo/issues/new?template=issue.yml) for this? (see https://github.com/pradyunsg/furo/issues/478 for an example of what those look like)

**Reply by PowerUser64** [reply]
Sure! Just made #515.
